### DISCUSSION TITLE
Can I turn off translation propagation for certain strings?

### DISCUSSION THREAD

**Discussion by luebbe** [WeblateOrg/weblate#6926] [Q&A] [answered]
Example:
"Current" in English may translate to "Aktuell"/"Actuel" or "Strom"/"Courant" in German/French depending on context and probably in most other languages as well.
After turning translation propagation on I have found no way to translate "Current->Courant" in one component and "Current->Actuel" in another component without the change in one component overwriting the translation in the other.

**Comment by nijel** [comment, marked as answer]
With translation propagation enabled you need to keep the key/context usage consistent between the components to avoid such issues.

**Comment by luebbe** [comment]
Thanks, so I have to turn translation propagation off, because I can't ensure that the context is identical between components.
Would this be worth a feature request: "Add a flag that turns translation propagation off for a particular ressource string"?

**Reply by nijel** [reply]
This is currently not easily possible - the flags are not stored in a way that would allow effective querying, so this would make the propagation really expensive operation.
